pulp/mps_lp.py

```diff
@@ -83,6 +86,12 @@ def readMPS(path, sense, dropConsNames=False):
                     mode = CORE_FILE_BOUNDS_MODE_NAME_GIVEN
                 else:
                     mode = CORE_FILE_BOUNDS_MODE_NO_NAME
+            elif parameters['sense'] is None and line[0] == CORE_FILE_OBJSENSE_MODE:
+                if len(line) > 1:
+                    parameters['sense'] = const.LpMaximize if line[1] == 'MAX'\
+                            else const.LpMinimize
```

Comment:
Aren't you missing a `mode=CORE_FILE_OBJSENSE_MODE` here too?

Reply:
Not in this case. This line is catching the case where you have the objective sense provided in the same line as the keyword `OBJSENSE`, such as
```
OBJSENSE MAX
```
In this case we don't need to enter the OBJSENSE mode.